modules/update-upgrading-cli.adoc

```diff
@@ -37,6 +37,13 @@ NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
 version   4.9.23     True        False         158m    Cluster version is 4.9.23
 ----
 
+. Set your channel to `stable-4.10`:
++
+[source,terminal]
+----
+$ oc adm upgrade channel stable-4.10
+----
+
 . Review the current update channel information and confirm that your channel
 is set to `stable-4.10`:
```

Comment:
going from 4.9 to 4.10, the main thing you want is the `-4.10` suffix.  `stable` is a reasonable prefix, but other prefixes would work too.

Reply:
Done!

Reply:
This sentence still says "confirm that your channel is set to `stable-4.10`, without allowing for the other supported 4.10 channels.

Reply:
@wking Instead of removing the "Review the current..." step, shall we rewrite it like this:

> Review the current update channel information:
> $ oc get clusterversion -o json|jq ".items[0].spec"
> 
> Example output
> {
>   "channel": "stable-4.10",
>   "clusterID": "990f7ab8-109b-4c95-8480-2bd1deec55ff"
> }
> 

I feel this step would ensure that admin is doing the right thing. What you say ?